docs/src/rippledAPIs.md.ejs

```diff
@@ -5,7 +5,7 @@ ripple-lib relies on [rippled APIs](https://ripple.com/build/rippled-apis/) for
 * Use `hasNextPage()` to determine whether a response has more pages. This is true when the response includes a [`marker` field](https://ripple.com/build/rippled-apis/#markers-and-pagination).
 * Use `requestNextPage()` to request the next page of data.
 
-When using rippled APIs, [specify XRP amounts in drops](https://ripple.com/build/rippled-apis/#specifying-currency-amounts). 1 XRP = 1000000 drops.
+When using rippled APIs, [specify XRP amounts in drops and timestamps as the number of seconds since the "Ripple Epoch"](https://developers.ripple.com/basic-data-types.html#specifying-currency-amounts).
```

Comment:
Consider linking the timestamps portion to https://developers.ripple.com/basic-data-types.html#specifying-time